experiments/scripts/eval_create_batch_configs.py

```diff
@@ -0,0 +1,43 @@
+import argparse
+import os
+
+import yaml
+from lm_eval import config
+
+CHECKPOINT_DIR = "checkpoint-final"
+
+
+def run(
+    config_path: str,
+    root_models_path: str,
+):
+    raw_config = config.load_config(config_path)
+
+    model_names = [
+        name
+        for name in os.listdir(root_models_path)
+        if os.path.isdir(os.path.join(root_models_path, name))
+    ]
+
+    for model_name in model_names:
+        raw_config[
+            "model_args"
+        ] = f"pretrained={root_models_path}/{model_name}/{CHECKPOINT_DIR}"
+        raw_config["wandb_run_name"] = model_name
+
+        new_config = open(f"{root_models_path}/{model_name}/eval_config.yml", "w")
```

Comment:
@jackapbutler  Do you think that because each f"{root_models_path}/{model_name}" is made by looping over a list of the dirs in root_models_path that f"{root_models_path}/{model_name}" must exist? Or you think there is a situation where this could fail?

Reply:
oh yes true, I forgot about that 😅